### ISSUE TITLE
Automatic Google Analytics event tracking page property is incorrect for HTML5Mode

### ISSUE THREAD

**Issue by wlingke** [luisfarzati/angulartics#356]
Currently, the GA Angulartics sets `eventOptions.page` using `page: properties.page || window.location.hash.substring(1),` 

The latter half (automatic part) is what I am referring to. If we are using HTML5Mode, this just returns a blank string.

**Comment by lunks** [comment]
Maybe we should strip the first char if it is a `#`?

**Comment by wlingke** [comment]
How about doing the same thing you do in page tracking? Inject $location or $state and retrieve the correct URL.

A second though off of the top of my head is to use `location.href` and just strip `location.origin` from the beginning.

URLs not using HTML5 mode will just have a "#" in front of it which is not unreasonable.

**Comment by luisfarzati** [comment]
Yes, I think the best is to send raw URLs just the way they are and, in any case, provide a way to configure "URL formatters" into Angulartics. But making this a default would impact current non-HTML5 mode users. Thoughts?
